### ISSUE TITLE
Using Seurat with multi sample CITE-seq data 

### ISSUE THREAD

**Issue by lg72cu** [satijalab/seurat#924]
Hi,

Thank you for the great tool and helpful tutorials!

I’m working on a project with 10X CITE-seq and scRNA-seq data. It is a little more complicated than the sample data in your tutorial “Using Seurat with multi-modal data” as it has two samples: before and after treatment. I’ve read another of your tutorials “Tutorial: Integrating stimulated vs. control PBMC datasets to learn cell-type specific responses” for performing comparative analyses. I’m wondering if you have any suggestions or experiences on how to integrate CITE-seq and scRNA-seq data from two samples to perform comparative analyses for both RNA and protein.

Thank you!
Lina

**Comment by satijalab** [comment]
Thanks for your question. Seurat v3 is designed to work flexibly across multiple assays - so you can easily find anchors using RNA or protein data, prior to performing comparative analysis for each modality.

**Comment by marencc** [comment]
I have a question regarding this issue. What would it be the proper way of analyzing the data. 
So far, I have followed your "Standard Workflow" for each "Control-Stimulated" data, then Integrating that. Afterwards, I've divided again in Control/Stimulated to add the corresponding protein data to each data set. Afterwards, I've proceeded to the "Using Seurat with multi-modal data" for each Control/Stim data set. But, now I was wodering if it's logical to now re-integrate using the protein expression levels as default assays? Or would it be sense-less?  Is it correct to integrate "Twice"?
